test/alternator/pytest.ini

```diff
@@ -2,3 +2,5 @@
 # pytest will look for one in our ancestor directories, and may find
 # something irrelevant. So we should have one here, even if empty.
 [pytest]
+filterwarnings =
```

Comment:
nit, the comment explaining why we have to have an empty `pytest.ini` above does not apply anymore.

Reply:
Right. Improved this (I hope) in the new version.